exercises/practice/binary-search-tree/tests/Tests.elm

```diff
@@ -0,0 +1,65 @@
+module Tests exposing (tests)
+
+import BinarySearchTree exposing (BinaryTree(..))
+import Expect
+import Test exposing (Test, describe, skip, test)
+
+
+tests : Test
+tests =
+    describe "BinarySearchTree"
+        [ -- skip <|
```

Comment:
These skips need to be restored I think

Reply:
Oops! Done in [e0a1763](https://github.com/exercism/elm/pull/609/commits/e0a1763ab98bbddf147416473a790c715834d324)